pyccel/naming/languagenameclashchecker.py

```diff
@@ -19,6 +19,11 @@ class LanguageNameClashChecker(metaclass = Singleton):
     """
     keywords = None
 
+    def __init__(self): #pylint: disable=useless-parent-delegation
+        # This __init__ function is required so the Singleton can detect a signature
+        super().__init__()
+
+
     def _get_collisionless_name(self, name, symbols):
```

Comment:
Too many blank lines to separate methods of a class. One is enough 😉

Reply:
Fixed [e0f8120](https://github.com/pyccel/pyccel/pull/2003/commits/e0f8120fd44b8d808bc17a1af4c6fdc920cf9a87)